runtime/tutor.txt

```diff
@@ -416,6 +415,296 @@ _________________________________________________________________
 
 
 
+
+=================================================================
+=                 3.5 SELECTING TO A CHARACTER                  =
+=================================================================
+
+ Press f<char> to select up to and including a character.
+ Press t<char> to select up to, but not including a character.
+ Press upper case F/T to do the same backwards.
+
+ 1. Move the cursor to the line below marked -->. Make sure your
+    cursor is put just after the arrow.
+ 2. Press f, followed by the character you want to select
+    to, in this case a square bracket: [.
+ 3. press d to delete your selection.
+ 4. Go to the end of the line and repeat with F].
+ 5. Move to the second line marked -->, just after the arrow.
+ 6. Use t and T to delete the dashes around the sentence
+
+ --> -----[Free this sentence of it's brackets!]-----
+ --> ------Free this sentence of it's dashes!------
+
+ Note: Unlike Vim, Helix doesn't limit these commands to just one
+       line. It'll just search for the character in the file.
+=================================================================
+=                        CHAPTER 3 RECAP                        =
+=================================================================
+
+ * Press C to copy the selection on the line below and Alt-C for
+   above.
+
+ * Press s to select all instances of a regex pattern inside
+   the current selection.
+
+ * Press semicolon ( ; ) to collapse selection.
+
+ * Press f / F to extend selection up to & including a character.
+
+ * Press t / T to extend selection until a character.
+
+
+
+
+
+
+
+=================================================================
+=                 4.1 COPYING AND PASTING TEXT                  =
+=================================================================
+
+ Press y to yank (copy) the selection.
+ Press p to paste the yanked selection after the cursor.
+ Press uppercase P to paste the yanked selection *before*
+ the cursor.
+
+ 1. Move the cursor to the line below marked -->.
+    Make sure your cursor is on the "b" of banana.
+ 2. Press w to select "banana" and y to yank it.
+ 3. Move to the space between "2" and "3" and press p to paste.
+ 4. Repeat between "3" and "4".
+
+ --> 1 banana 2 3 4
+     1 banana 2 banana 3 banana 4
+
+ Note: Whenever you delete or change text, Helix will copy the
+       altered text. Use alt-d/c instead to avoid this.
+ Note: Helix doesn't share the system clipboard by default. Press
+       space-y/p to yank/paste on your computers main clipboard.
```

Comment:
```suggestion
       space-y/p to yank/paste on your computer's main clipboard.
```

Reply:
Thanks. I did this manually rather than use the "commit suggestion" because that's when things went sideways with git before, but it's fixed.